tools/ports/bzip2.py

```diff
@@ -7,19 +7,19 @@
 import shutil
 
 VERSION = '1.0.6'
-HASH = '00ace5438cfa0c577e5f578d8a808613187eff5217c35164ffe044fbafdfec9e98f4192c02a7d67e01e5a5ccced630583ad1003c37697219b0f147343a3fdd12'
+HASH = '512cbfde5144067f677496452f3335e9368fd5d7564899cb49e77847b9ae7dca598218276637cbf5ec524523be1e8ace4ad36a148ef7f4badf3f6d5a002a4bb2'
 
 
 def get(ports, settings, shared):
   if settings.USE_BZIP2 != 1:
     return []
 
-  ports.fetch_project('bzip2', 'https://sourceware.org/pub/bzip2/bzip2-1.0.6.tar.gz', 'bzip2-1.0.6', sha512hash=HASH)
+  ports.fetch_project('bzip2', 'https://github.com/emscripten-ports/bzip2/archive/' + VERSION + '.zip', 'bzip2-' + VERSION, sha512hash=HASH)
 
   def create():
     ports.clear_project_build('bzip2')
 
-    source_path = os.path.join(ports.get_dir(), 'bzip2', 'bzip2-1.0.6')
+    source_path = os.path.join(ports.get_dir(), 'bzip2', 'bzip2-' + VERSION)
```

Comment:
This change looks unrelated - accidental, or do we want to land this in?

Reply:
That's already landed on master, I added it here to verify Ci is green.